### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by rhagarty**

### CODE REVIEW

**Comment by rhagarty** [comment]
Don't merge yet - need to verify Nimbix registration process

File: README.md [hunk 1]
```diff
@@ -47,7 +47,24 @@ described in detail below.
 
 ## 1. Register for a Nimbix Cloud Platform account
 
-Register for Power AI on the [Nimbix Cloud](https://platform.jarvice.com/login-page).
+IBM has partered with Nimbix to provide journey developers a trial
+account that provides 10 hours of free processing time on the Power AI
```
Comment (stevemart) [inline README.md]: PowerAI, one word

File: README.md [hunk 2]
```diff
@@ -47,7 +47,24 @@ described in detail below.
...
+platform.
+
+The sign-ip process is as follows:
```
Comment (stevemart) [inline README.md]: sign-up*

File: README.md [hunk 3]
```diff
@@ -47,7 +47,24 @@ described in detail below.
...
+
+1) Go to the [IBM Congnitive Journey Demonstration Registration](https://www.nimbix.net/cognitive-journey)
```
Comment (stevemart) [inline README.md]: "Demonstration Registration" sounds funny, can we remove "Demonstration" or shorten it to "Demo"?

Comment (markstur) [inline README.md]: Typo in "Congnitive"

**Comment by markstur** [comment]
a couple of typos.  I'll try the registration now.

File: README.md (hunk 1 above)
Comment (rhagarty) [inline README.md]: good catch

File: README.md (hunk 3 above)
Comment (rhagarty) [inline README.md]: Agree... and thanks for finding the typos

**Comment by stevemart** [comment]
So, the changes look good to me, but I can't get the registration process to complete, it just hangs for me.

**Comment by rhagarty** [comment]
No longer needed

**Pull request closed without merging**